apps/microsoft-teams/frontend/src/components/config/ChannelSelection/ChannelSelection.tsx

```diff
@@ -51,12 +64,14 @@ const ChannelSelection = (props: Props) => {
             icon={<EditIcon />}
             onClick={openChannelSelectionModal}
             aria-label="Change selected channel"
+            isLoading={areChannelsLoading}
           />
         </TextInput.Group>
       ) : (
         <AddButton
           buttonCopy={channelSelection.addButton}
           handleClick={openChannelSelectionModal}
+          isLoading={areChannelsLoading}
```

Comment:
yeah I thought the same thing while implementing it, but on first glance found it looked kind of funny while playing around with it, though I do think it would further prevent the user from clicking it. I am happy to disable for now and see how we like it?

Reply:
Awesome, let's go with Emily's call of not having the disabled state, loading is enough to indicate that the button isn't actionable